test/integration/JestTestRunner.it.ts

```diff
@@ -138,9 +150,9 @@ describe('JestTestRunner', function () {
         before(() => {
             const testRunnerOptions = {
                 files: [
-                    { path: 'testResources/sampleProject/src/Add.js', mutated: true, included: true },
-                    { path: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false, included: true },
-                    { path: 'testResources/sampleProject/src/Error.js', mutated: false, included: false }],
+                    file({ name: 'testResources/sampleProject/src/Add.js', mutated: true }),
+                    file({ name: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false }),
+                    file({ name: 'testResources/sampleProject/src/Error.js', mutated: false })],
```

Comment:
I think this one should have `included: false`, if I'm not mistaken?

Reply:
You are definitely correct